### ISSUE TITLE
convert DUAs from txt to html

### ISSUE THREAD

**Issue by robertoostenveld** [donders-research-data-management/rdm-configurable-content-donders#7]
This relates to issue #6

**Comment by robertoostenveld** [comment]
Hi @hurngchunlee, all DUAs are now present as txt and as html. I think the txt version can be removed, right?

**Comment by hurngchunlee** [comment]
I don't think so, I see this index file still refers to .txt files: https://github.com/donders-research-data-management/rdm-ontology/blob/master/dua/index.json

**Comment by hurngchunlee** [comment]
If you remove .txt files, you should also change the index.json to refer to .html files

**Comment by robertoostenveld** [comment]
are the txt files then included anywhere by the java CMS?

**Comment by hurngchunlee** [comment]
Java code should get the file from the index.json ...

**Comment by robertoostenveld** [comment]
The html content of https://data.donders.ru.nl/datauseagreement/PD is now

```
<div id="content" class="container">
<div class="page-header">
<h1>Data use agreement PD </h1>
</div>

<div>
<pre>This work is free of known copyright restrictions.
</pre>
</div>

</div>
```

which indeed suggests that the content is from the txt, that it is not formatted as-is, but wrapped in <pre> and that the title is added by the Java code. 

So at this moment we cannot remove the txt files and further improve the display in the html, since the Java code is preventing us from a proper display...

**Comment by robertoostenveld** [comment]
I have added it here https://radboud.atlassian.net/browse/RDM-1326

**Comment by robertoostenveld** [comment]
this has been resolved and merged in #14
